### ISSUE TITLE
Добавить возможность выбора адреса для заливки прошивки

### ISSUE THREAD

**Issue by lomalkin** [Flipper-Zero/flipperzero-remote-testbench-front#7]

**Comment by imShara** [comment]
Предлагаю брать из названия файла, чтобы случайно не то не туда не залить

**Comment by lomalkin** [comment]
> Предлагаю брать из названия файла, чтобы случайно не то не туда не залить

Нет, надо точно дать возможность явно указывать. Поговорю с дизайнером как впихнуть это в шаблон.

**Comment by imShara** [comment]
> Нет, надо точно дать возможность явно указывать. Поговорю с дизайнером как впихнуть это в шаблон.

Диалогом, как ещё

**Comment by lomalkin** [comment]
Адрес должен запоминаться, хотябы до перезагрузки страницы. Это на случай загрузки нескольких прошивок подряд.

Дефолтный адрес "0x08000000" (передается строкой).
